source/api_cc/include/DeepPot.h

```diff
@@ -472,6 +481,14 @@ class DeepPotModelDevi {
     assert(inited);
     return ntypes;
   };
+  /**
+   * @brief Get the number of types with spin.
+   * @return The number of types with spin.
+   **/
+  int numb_types_spin() const {
```

Comment:
This new public method is not tested.

Reply:
Could you give me an example about how to test methods?

Reply:
https://github.com/deepmodeling/deepmd-kit/tree/devel/source/api_cc/tests

Reply:
Could you offer us a workflow for a new test_*.cc in this directory? We don't have much experience in GTest framework.

Reply:
OK.